src/clr-angular/data/datagrid/datagrid-row.ts

```diff
@@ -213,9 +247,16 @@ export class ClrDatagridRow<T = any> implements AfterContentInit, AfterViewInit
 
   ngOnInit() {
     this.wrappedInjector = new HostWrapper(WrappedRow, this.vcr);
+    this.oldHeight = this.getCurrentHeight();
```

Comment:
This is one should be moved to `ngAfterViewInit` where the view would be settled.

Reply:
gone in rework